src/Components/DetailedStockView.js

```diff
@@ -25,13 +27,17 @@ function DetailedStockView(props) {
   }
 
   const currentStock = props.currentStock;
+  const loggedUsername = props.loggedUsername;
+  const loggedIn = props.loggedIn;
   console.log(props);
   return (
     <div className='detailed-stock-view'>
       <Container>
         <h1 className="stock-header"> {currentStock.ticker} </h1>
         <StyledButton onClick={closeView} className="exit-view-button" variant="contained"> <b> X </b></StyledButton>
         <StyledButton className="add-watchlist-button" variant="contained"> <b> (+) Watchlist </b></StyledButton>
+        <SellPrompt currentStock= { currentStock } loggedUsername = {loggedUsername} loggedIn = {loggedIn}> </SellPrompt>
+        <BuyPrompt currentStock= { currentStock } loggedUsername = {loggedUsername} loggedIn = {loggedIn}> </BuyPrompt>
```

Comment:
Do you want to pass Logged in? Back-end doesn't take into account logged in

Reply:
need for now, will be changed when refactoring codebase